corehq/apps/domain/models.py

```diff
@@ -332,7 +332,7 @@ class Domain(QuickCachedDocumentMixin, BlobMixin, Document, SnapshotMixin):
     location_restriction_for_users = BooleanProperty(default=False)
     usercase_enabled = BooleanProperty(default=False)
     hipaa_compliant = BooleanProperty(default=False)
-    use_sql_backend = BooleanProperty(default=False)
+    use_sql_backend = BooleanProperty(default=True)
```

Comment:
It'd be a trivial thing to run too.  This should run the wrap command for all domains and save any which change in the process:

```python
from corehq.util.couch import iter_update, DocUpdate
from corehq.apps.domain.models import Domain

def ensure_wrapped(doc):
    return DocUpdate(doc=Domain.wrap(doc))

iter_update(Domain.get_db(), ensure_wrapped, Domain.get_all_ids())
```

Could then also remove most of the stuff that's accumulated in `wrap` over the years.  On the plus side, we probably only need to worry about prod, india, and staging, since non SaaS environments are all relatively new and don't have many domains.